frontend/src/v4/modules/issues/issues.sagas.ts

```diff
@@ -80,6 +90,14 @@ function* fetchIssues({teamspace, modelId, revision}) {
 }
 
 function* fetchIssue({teamspace, modelId, issueId}) {
+	yield waitForAddons();
+	const issuesEnabled = yield select(selectIssuesEnabled);
+
+	if (!issuesEnabled) {
```

Comment:
Is this needed for this saga? At what point in the app do we call `fetchIssue()` that hasn't already been disabled by the check in `fetchIssues()`?

Reply:
yeah, but it doesnt hurt to have it there also.